### ISSUE TITLE
Bug: XAML Controls Gallery: Accessibility:[MAS4.2.1]:[Narrator]:In SplitView card, narrator is not announcing the selected option and changes after activating options.

### ISSUE THREAD

**Issue by v-amnat** [microsoft/Xaml-Controls-Gallery#638]
**Tag**
Accessibility, A11yCT, A11yMAS, A11yMediumImpact, MAS4.2.1, Narrator, Sev3, XAML Controls Gallery (Debug), Win32, Wipro

**Hardware and Build Details:**
UWP or Desktop?: Desktop
Microsoft OS: 21313.1000
App: XAML Controls Gallery (Debug)(Version 2.1.0.0)

**Repro-Steps:**
Launch the XAML Controls Gallery app
Navigate to Layout tab and activate it.
Now navigate to the SplitView card and activate it.
Now activate controls(People, Globe, Message etc,.) in Pane Content group and observe.
Respective messages are shown in SplitView Content group but narrator is not announcing those messages instantly. 
Narrator is not announcing selected option as selected.

**Actual Result:**
In SplitView card, Narrator is not announcing message instantly in SplitView content when activating controls in Pane content.
 Narrator is also not announcing the selected state for the option.

Note: Selected option under Pane Content group is not highlighted and kb user may feel difficulty to identify the selected option.

**Expected Result:**
 In SplitView card, Narrator should announce message instantly in SplitView content when activating controls in Pane content.
Selected option should be highlighted with the blue colour and narrator should announce the selected option as selected.

**User Impact:**
People with Visual impairments are not aware of the messages displayed in the SplitView content when activating controls in Pane content and user is not aware of the selected option.

**Attachment**
[Narrator-SplitView-SelectionState_edit.zip](https://github.com/microsoft/microsoft-ui-xaml/files/6021744/Narrator-SplitView-SelectionState_edit.zip)

**Comment by StephenLPeters** [comment]
@stmoy FYI

**Comment by YuliKl** [comment]
Not a regression. The SplitView sample needs to be reworked as it predates NavigationView and feels redundant with that control.

**Comment by chingucoding** [comment]
This sample has been causing more pain now that it probably is worth, especially since it's not that helpful since there is NavigationView now. Should we remove it then instead of trying to fix all the bugs accompanied by it?

**Comment by stmoy** [comment]
SplitView is still a legitimate control to use and has value beyond the NavigationView scenario. Agree with Yulia that reworking the sample probably makes sense.

I was thinking maybe we could repurpose it to be something like an "About Us" pane or something. I believe I have seen that usage in other places, might be appropriate here too.

**Comment by chingucoding** [comment]
> I was thinking maybe we could repurpose it to be something like an "About Us" pane or something. I believe I have seen that usage in other places, might be appropriate here too.

I'm not sure if I'm following you here. How would this sample look? Would it just be "one sided"?

**Comment by stmoy** [comment]
I was thinking something like the About Us flyout found in the Mail app: 

![image](https://user-images.githubusercontent.com/25991996/109194768-9cfe5480-774e-11eb-81c3-8c672ca88796.png)

**Comment by chingucoding** [comment]
Ah I see, yes we can definitely do that. Should the sample still be "interactive"? Can we cut down on the API to simplify the sample a bit?

Also re "Should we remove it then instead of trying to fix all the bugs accompanied by it?", I though that we had multiple samples of the controls so removing one wouldn't be a big deal. However removing the only sample without replacement is definitely not what I had in mind.

**Comment by chigy** [comment]
@anawishnoff fixed this
